src/Entity/IdeasWorkshop/EnabledInterface.php

```diff
@@ -0,0 +1,7 @@
+<?php
+
+namespace AppBundle\Entity\IdeasWorkshop;
```

Comment:
what about moving it to `AppBundle\Entity`? This interface can be used by other entities

Reply:
Done.